### DISCUSSION TITLE
Restoring project is stuck.

### DISCUSSION THREAD

**Discussion by Ansh-Rathod** [supabase/supabase#15216] [General]
the project is stuck here for the last 4 days. 

<img width="1277" alt="Screenshot 2023-06-20 at 7 51 30 PM" src="https://github.com/supabase/supabase/assets/67627096/07fe7a85-a181-41c8-978c-914bc0bf2ce0">

**Comment by nyancodeid** [comment]
Same here, i have no idea what's going on :confused:

**Reply by RGeorge-IN** [reply]
Same here. It's been 24+ hrs.

**Comment by luizfx22** [comment]
I've facing the same issue, it's stuck on restoring for 26 hours now

**Reply by GaryAustin1** [reply]
Contacting support is the only way to get it unstuck.

**Reply by luizfx22** [reply]
I've contacted, waiting for response! Thanks!
